data/seeds/18-contact_preferences.js

```diff
@@ -20,4 +20,4 @@ exports.seed = function (knex) {
         },
       ]);
     });
-};
```

Comment:
Looks like there is an eslint issue here. I think we just need to add a blank line at the end of this code.